tools/x-docs/src/docs/guides/components/cookbook/testing.md

```diff
@@ -0,0 +1,43 @@
+# Testing components
+
+Defining [stories](../setup/#writing-tests--demos) from your components, as well as including them in Storybook, will generate Jest snapshot tests for each story. Your stories should include as many of your component's use cases as possible, so that they're covered by the tests.
```

Comment:
"the tests" => "these tests"?

Reply:
i was going for `the tests` in abstract, not any particular tests, but x-dash's entire test suite